src/openzeppelin/account/interface.cairo

```diff
@@ -2,8 +2,8 @@ use array::ArrayTrait;
 use array::SpanTrait;
 use starknet::ContractAddress;
 
-const IACCOUNT_ID: u32 = 0xa66bd575_u32;
-const ERC1271_VALIDATED: u32 = 0x1626ba7e_u32;
+const IACCOUNT_ID: felt252 = 0x36c738c1c375b993078fe6b517d477e5a3c9b104e40c04662c4bdd3e2f5fa4a;
+const IERC1271_VALIDATED: u32 = 0x1626ba7e_u32;
```

Comment:
why do we need to add `I` if it's not acknowledged as an interface ID but a magic value instead?